src/Raven.Server/Documents/TimeSeries/TimeSeriesStorage.cs

```diff
@@ -235,6 +235,32 @@ private long PurgeSegments(long upto, DocumentsOperationContext context, Collect
             return deletedCount;
         }
 
+        public long GetNumberOfTimeSeriesDeletedRanges(DocumentsOperationContext context)
+        {
+
+            var fstIndex = DeleteRangesSchema.FixedSizeIndexes[AllDeletedRangesEtagSlice];
+            var fst = context.Transaction.InnerTransaction.FixedTreeFor(fstIndex.Name, sizeof(long));
+            return fst.NumberOfEntries;
+        }
+
+        public long GetNumberOfTimeSeriesPendingDeletionSegments(DocumentsOperationContext context)
+        {
+            var timeSeriesPurgeSegments = 0l;
```

Comment:
Let's remove warning here `The 'l' suffix is easily confused with the digit '1' -- use 'L' for clarity`